keyform/static/keyform/js/request.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() {
+    $('#cancel').css("display", 'none');
+    $('#submit').css("display", 'none')
+    var div = document.getElementById('comments');
+    $('#add').click(function() {
+        $('#add').css("display", 'none');
+        $('#cancel').css("display", 'inline-block');
+        $('#submit').css("display", 'inline-block')
```

Comment:
Alternatively you could toggle a class, that way if you want to do more than just one css thing all you need to do is add to the class
```javascript
$('#submit').toggleClass('my-awesome-css-class');
```
And if you want to get fancy you can add a second argument to tell jquery you want to add/remove the clas